rustfmt.toml

```diff
@@ -4,3 +4,82 @@
 # be picked up automatically).
 version = "Two"
 use_small_heuristics = "Max"
+
+# by default we ignore everything in the repository
+# tidy only checks files which are not ignored, each entry follows gitignore style
+ignore = [
+    # remove directories below, or opt out their subdirectories, as they are formatted
+    "src/bootstrap/",
+    "src/build_helper/",
+    "src/liballoc/",
+    "src/libarena/",
+    "src/libcore/",
+    "src/libfmt_macros/",
+    "src/libgraphviz/",
+    "src/libpanic_abort/",
+    "src/libpanic_unwind/",
+    "src/libproc_macro/",
+    "src/libprofiler_builtins/",
+    "src/librustc/",
+    "src/librustc_apfloat/",
+    "src/librustc_asan/",
+    "src/librustc_codegen_llvm/",
+    "src/librustc_codegen_ssa/",
+    "src/librustc_codegen_utils/",
+    "src/librustc_data_structures/",
+    "src/librustc_driver/",
+    "src/librustc_errors/",
+    "src/librustc_incremental/",
+    "src/librustc_index/",
+    "src/librustc_interface/",
+    "src/librustc_lexer/",
+    "src/librustc_lint/",
+    "src/librustc_llvm/",
+    "src/librustc_lsan/",
+    "src/librustc_macros/",
+    "src/librustc_metadata/",
+    "src/librustc_mir/",
+    "src/librustc_msan/",
+    "src/librustc_passes/",
+    "src/librustc_plugin/",
+    "src/librustc_privacy/",
+    "src/librustc_resolve/",
+    "src/librustc_save_analysis/",
+    "src/librustc_target/",
+    "src/librustc_traits/",
+    "src/librustc_tsan/",
+    "src/librustc_typeck/",
+    "src/librustdoc/",
+    "src/libserialize/",
+    "src/libstd/",
+    "src/libsyntax/",
+    "src/libsyntax_expand/",
+    "src/libsyntax_ext/",
+    "src/libsyntax_pos/",
+    "src/libterm/",
+    "src/libtest/",
+    "src/libunwind/",
+    "src/rtstartup/",
+    "src/rustc/",
+    "src/rustllvm/",
+    "src/stdarch/",
```

Comment:
This is a submodule, so I think it should in the second section after the newline below.